### PR TITLE
fix: update mew-notification by using v-expansion-panels

### PR DESCRIPTION
**Pull request by hsurf22**
update mew-notification by using v-expansion-panels
https://app.asana.com/0/1125971615701970/1202297334683712

### CODE REVIEW

File: src/components/MewNotification/MewNotification.vue [hunk 1]
```diff
@@ -1,192 +1,342 @@
...
+                  <!-- ===================================================================================== -->
+                  <div
+                    v-if="
+                      $vuetify.breakpoint.smAndUp &&
+                      !notification.read &&
+                      showIndicator
+                    "
+                    :class="[
+                      getClasses(notification.status.value.toLowerCase()),
+                      'indicator',
```
Comment (iluvboba) [inline src/components/MewNotification/MewNotification.vue:35]: this indicator styles isn't showing up anymore

File: src/components/MewNotification/MewNotification.vue [hunk 2]
```diff
@@ -1,192 +1,342 @@
...
+            </div>
+          </v-expansion-panel-content>
+        </v-expansion-panel>
+      </v-expansion-panels>
+    </div>
+
+    <!-- ===================================================================================== -->
+    <!-- Mew Notifications -->
+    <!-- ===================================================================================== -->
+    <div
```
Comment (iluvboba) [inline src/components/MewNotification/MewNotification.vue:182]: remove unused code

**Comment by iluvboba** [comment]
also clicking on the notification should remove the outside border at least that is how it is currently working

File: src/components/MewNotification/MewNotification.vue (hunk 2 above)
Comment (hsurf22) [reply src/components/MewNotification/MewNotification.vue]: unused code removed

File: src/components/MewNotification/MewNotification.vue (hunk 1 above)
Comment (hsurf22) [reply src/components/MewNotification/MewNotification.vue:35]: fixed by adjusting content width

Comment (iluvboba) [reply src/components/MewNotification/MewNotification.vue:35]: oh i meant its not getting the styles from the `.indicator` class for some reason

Comment (iluvboba) [reply src/components/MewNotification/MewNotification.vue:35]: so its a square instead of a circle

Comment (hsurf22) [reply src/components/MewNotification/MewNotification.vue:41]: fixed the indicator shape

**Comment by hsurf22** [comment]
also clicking on the notification should remove the outside border at least that is how it is currently working


https://user-images.githubusercontent.com/9893774/170615227-a374a843-07bc-40ae-a767-294b107cc48f.mp4

**Review by iluvboba — APPROVED** [review]

**Pull request merged**